src/components/shift-schedule/ScheduleCell.tsx

```diff
@@ -0,0 +1,62 @@
+import { ScheduleCellDialog } from '@/components/shift-schedule/ScheduleCellDialog';
+import { Dialog, DialogContent, DialogTrigger } from '@/components/ui/Dialog';
+import { TableCell } from '@/components/ui/Table';
+import { cx } from '@/lib/utils';
+import { UserIcon, UsersIcon } from 'lucide-react';
+import { useTranslations } from 'next-intl';
+
+type ScheduleCellProps = {
+  tDialog: {
+    day: string;
+    time: string;
+  };
+  members: {
+    name: string;
+  }[];
+};
+
+function ScheduleCell({ tDialog, members }: ScheduleCellProps) {
+  const t = useTranslations('shiftSchedule.scheduleTable.scheduleCell');
+
+  return (
+    <TableCell className='h-20 min-w-52 border p-1.5'>
+      <Dialog>
+        <DialogTrigger asChild>
+          <button
+            type='button'
+            className={cx(
+              'flex size-full gap-2 rounded-md p-3 text-left',
+              members.length === 0
+                ? 'bg-accent/50 text-accent-foreground hover:bg-accent dark:bg-accent/40 dark:hover:bg-accent/60'
+                : 'bg-foreground/20 hover:bg-foreground/25',
+            )}
+          >
+            {/* Icon displaying amount of people on shift */}
+            {members.length === 1 ? (
+              <UserIcon className='size-7' />
+            ) : members.length > 1 ? (
+              <UsersIcon className='size-7' />
+            ) : (
+              <></>
+            )}
```

Comment:
```suggestion
            {members.length === 1 ? (
              <UserIcon className='size-7' />
            ) : (
              members.length > 1 && <UsersIcon className='size-7' />
            )}
```
If the initial conditional is false, the second option will not return anything if there are no members on this shift.